pr2eus/robot-interface.l

```diff
@@ -892,4 +892,15 @@
           (nreverse tms))
     ))
 
+(defun make-robot-interface-from-name (name &rest args)
+  "make a robot model from string: (make-robot-model \"pr2\")"
+  (let ((name (string-downcase name)))
+    (let ((klasses (flatten (send robot-interface :hierarchy))))
+      (dolist (klass klasses)
+        (let ((klass-name (string-downcase (send klass :name))))
+        (if (or (string= klass-name name)
```

Comment:
I'd like to keep searching rules simple.
Why isn't it sufficient to search [robot-name]-interface?
How about to define function like this?
```
(let ((klass (read-from-string (format nil "~A-interface" robot-name))))
  (if (and (boundp klass) (subclassp (eval klass) robot-interface))
      (instance* (eval klass) :init args)
    (error "No such subclass of robot-interface : ~A~%" klass)))
```